hip-lang-config.cmake.in

```diff
@@ -65,18 +65,81 @@ find_dependency(amd_comgr)
 
 include( "${CMAKE_CURRENT_LIST_DIR}/hip-lang-targets.cmake" )
 
+
+# Approach:Find LLVM directly using find_dependency()
+# IF LLVM_Successfully found, Check for VALID LLVM installation
+# using LLVM package version major no & amdgcn target check of clang binary
+# available in the LLVM install path found.
+# On Failure: If VALID LLVM installation is not found
+# Continued with old approach to use find_path() to search for LLVM include path
+# in predefined relative search Directories
+find_dependency(LLVM)
```

Comment:
You should not do `find_dependency(LLVM)` as this might find a different llvm than the one being used by `CMAKE_HIP_COMPILER`.